webapp/src/ts/modals/check-date/check-date.component.html

```diff
@@ -0,0 +1,22 @@
+<mm-modal
+  id="'check-date'"
+  [titleKey]="'date.incorrect.title'"
+  [submitKey]="'date.incorrect.confirm'"
+  [cancelKey]="''"
```

Comment:
Continuing from the comment above, setting this as an empty string is strange. The translator will probably try to translate it (and who knows....). Wouldn't it be more clearer if we set it to boolean `false` when we don't want a cancel button?